### PR TITLE
Cycling through All Clients

### PR DESCRIPTION
**Pull request by ksharizard**
This PR allows to cycle through all the clients available on all tags (and screens?). It is disabled by default. Fairly new to lua so would appreciate any reviews.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -42,6 +42,7 @@ _M.settings = {
 	client_opacity_value = 0.5,
 
 	cycle_raise_client = true,
+  cycle_all_clients  = true,
```
Comment (troglobit) [inline init.lua]: You say in the PR description; "It is disabled by default." but here it is clearly _enabled_.  I think your description is a good default, so please change this and update the README as well.

Also, you seem to use a different tab width setting, so this line does not have the same indentation as surrounding lines.

File: init.lua [hunk 2]
```diff
@@ -498,6 +499,7 @@ function _M.switch(dir, mod_key1, release_key, mod_key2, key_switch)
 						-- raise chosen client on top of all
 						c = _M.altTabTable[_M.altTabIndex].client
 						c:raise()
+            c:jump_to()
```
Comment (troglobit) [inline init.lua]: Here too, the indentation is off compared to surrounding lines.

**Review by troglobit — CHANGES_REQUESTED** [review]
The idea is good, but you need to fix the comments I made, and please rebase and squash your changeset to a single commit.

File: init.lua (hunk 1 above)
Comment (ksharizard) [reply init.lua]: Made false as default option in both settings and README.

File: init.lua (hunk 2 above)
Comment (ksharizard) [reply init.lua]: Fixed the indentation.

**Comment by troglobit** [comment]
Thanks, great work!

**Pull request merged**